modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -12,6 +12,7 @@ export class ModelBuilderPage {
 	readonly addObjectFieldButton: Locator;
 	readonly createNewObjectDefinitionButton: Locator;
 	readonly deleteObjectDefinitionOption: Locator;
+	readonly deleteTrashButton: Locator;
```

Comment:
Hey @rafaellajordao, could you sort this Locator?

Reply:
fixed!